solutions/USACO open18-familytree.mdx

```diff
@@ -0,0 +1,125 @@
+---
+id: usaco-833
+source: USACO Bronze 2018 Open
+title: Family Tree
+author: Danh Ta Chi Thanh
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_family_bronze_open18.html)
+
+## Alternative Solution - BFS
+

```

Comment:
```suggestion

## Solution

<IncompleteSection>

Add Intended Solution

</IncompleteSection>

```

we can do this as a cop-out

Reply:
I thought the main solution is in the **official analysis**?

Reply:
Yes. Maybe just have alternate code or smth? I mean ofc we can just leave it blank and only have alternate.

Reply:
I cannot english. What I mean:
1) We can either mention the main solution (and have some content under it, perhaps just an implementation), or
2) We can not mention the main solution and only have the alternate

Reply:
I think we can have something like this:

```

## Main solution (for Silver, Bronze)

paste the official analysis link here

## Alternative solution (for Gold+)

Insert my solution here

```

Reply:
```
## Solution
```
and
```
## Alternate Solution
```
are sufficient

Typically we separate main solution and editorial even if they are the same though... Maybe we can just say "check official editorial above" or smth